draft-ietf-masque-connect-ip.md

```diff
@@ -174,24 +210,29 @@ optional, and have default values if not included.
 The defined variables are:
 
 target:
-: The variable "target" contains a hostname or IP address of a specific host to
-which the client wants to proxy packets. If the "target" variable is not
-specified, the client is requesting to communicate with any allowable host. If
+: The variable "target" contains a DNS hostname (reg-name), IPv6 literal
+(IPv6address) or IPv4 literal (IPv4address) address ({{URI}} syntax elements
+within parenthis) of a specific host to which the client wants to proxy
+packets. If the "target" variable is not specified or its value is "\*", the
```

Comment:
```suggestion
packets. If the "target" variable is not specified or its value is "*", the
```

Reply:
Ah thanks, it seemed to work for the one that was missing the backslash in the GitHub Pages generated version, but it doesn't hurt to have the backslash so might as well keep it